.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,50 @@
+name: Bug report
+description: Report any issues that you have found with the Sweyer app. Please check open issues first, in case it has already been reported.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time and filling out this bug report!
+
+        Please [check open issues](https://github.com/nt4f04uNd/sweyer/issues) first, in case it has already been reported.
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: Please attach screenshots, videos or logs if you can.
+      placeholder: Describe what you did to trigger the bug.
+      value: |
+        1. Where are you starting? What can you see?
+        2. What do you click?
+        3. More steps…
+    validations:
+      required: true
+  - type: textarea
+    id: result
+    attributes:
+      label: Outcome
+      placeholder: Tell us what went wrong
+      value: |
+        #### What did you expect?
+
+        #### What happened instead?
+        
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Sweyer version
+      description: You can find the version information in App Menu -> Settings at the bottom. If you compiled the app yourself, please specify the commit here.
+      placeholder: e.g. 1.0.7
+    validations:
+      required: true
+  - type: input
+    id: android-version
+    attributes:
+      label: Android version
+      description: The Andorid version of your device. You can leave this empty if you are sure that it's not important, if you are unsure please specify. 
```

Comment:
Let's remove "You can leave this empty if you are sure that it's not important, if you are unsure please specify." part since if we are adding it here, we would want people to add it here, and this is explanation is a bit specific. Leaving this as non-required is sufficient for people to undesrtand they can leave this empty.